docs/badges_buttons.md

```diff
@@ -215,7 +215,7 @@ Material Design icons come as several flavors. Each flavor represents a differen
 
 Not all icons are available for each flavor, but most are. Instead of displaying the 10660+ icons here, you are encouraged to browse the available icons from the [Material Design Icons' showcase](https://fonts.google.com/icons) hosted by Google.
 
-- A regular icon: {material-regular}`thumbs_up_down`, some more text
+- A regular icon: {material-regular}`data_exploration`, some more text
```

Comment:
I'm ok with this. Originally, I just wanted to demonstrate the differences between flavors of the same icon.

Reply:
👍